tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/oidc/JMAPOidcModule.java

```diff
@@ -0,0 +1,63 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail.james.jmap.oidc;
+
+import java.net.URL;
+
+import org.apache.commons.configuration2.Configuration;
+import org.apache.james.jmap.http.AuthenticationStrategy;
+import org.apache.james.jwt.introspection.IntrospectionEndpoint;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.multibindings.Multibinder;
+import com.google.inject.name.Named;
+import com.linagora.tmail.james.jmap.model.Aud;
+
+public class JMAPOidcModule extends AbstractModule {
+    @Override
+    protected void configure() {
+        Multibinder.newSetBinder(binder(), AuthenticationStrategy.class)
+            .addBinding()
+            .to(OidcAuthenticationStrategy.class);
```

Comment:
IMO we should let this done by configuration and should avoid hard coding it.

Do we want to be able to start Twake mail without OIDC?